<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PracticOOPClass</class>
 <widget class="QWidget" name="PracticOOPClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PracticOOP</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>341</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>211</width>
     <height>251</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QSplitter" name="splitter_2">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLineEdit" name="tasklineEdit"/>
    <widget class="QPushButton" name="addpushButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removepushButton">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startpushButton">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="donepushButton">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="exitpushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>410</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PracticOOP.qrc"/>
 </resources>
 <connections/>
</ui>
